<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QStatusBar {
color: #FF0000;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>单文件</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="tsPathEdit">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>ts文件路径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tsLookBtn">
             <property name="text">
              <string>浏览</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="excelPathEdit">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>excel文件路径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="excelLookBtn">
             <property name="text">
              <string>浏览</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>翻译所在列</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="transSpinBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>120</width>
               <height>0</height>
              </size>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="value">
              <number>3</number>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="tsImportBtn">
             <property name="text">
              <string>ts导入</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="generateBtn">
             <property name="text">
              <string>excel生成</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tsUpdateBtn">
             <property name="text">
              <string>译文写入</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>文件夹</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLineEdit" name="tsDirEdit">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>ts目录路径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tsDirLookBtn">
             <property name="text">
              <string>浏览</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="excelDirEdit">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>excel文件路径</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="excelDirBtn">
             <property name="text">
              <string>浏览</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="clearPl">
             <property name="text">
              <string>清空批量翻译队列</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="generateBtn_2">
             <property name="text">
              <string>批量excel生成</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="tsUpdateBtn_2">
             <property name="text">
              <string>批量译文写入</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="plTrans">
             <property name="text">
              <string>生成+翻译+导入(一键完成所有)</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>剩余翻译文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_fileCount">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>百度翻译(翻译内容为excel第3列)</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="youdaoAppIdLabel">
           <property name="text">
            <string>应用ID</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="4">
          <widget class="QLineEdit" name="youdaoAppIdlineEdit">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="youdaoKeyLabel">
           <property name="text">
            <string>应用密钥</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="4">
          <widget class="QLineEdit" name="youdaoKeylineEdit">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="youdaoDstLabel">
           <property name="text">
            <string>目标语言</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1" colspan="2">
          <widget class="QComboBox" name="comboBox"/>
         </item>
         <item row="2" column="3">
          <widget class="QLabel" name="youdaoDstLabel_2">
           <property name="text">
            <string>其他语言查询官方文档填写代码</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QLineEdit" name="otherLineEdit"/>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="translateBtn">
           <property name="text">
            <string>百度翻译</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2" colspan="3">
          <widget class="QLabel" name="youdaoTipLabel">
           <property name="text">
            <string>翻译完成，点击上面的excel生成到翻译所在列，点击译文写入输出到ts文件</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="5">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>单个导入请按照ts导入-&gt;excel生成(加入程序缓存)-&gt;开始翻译[在下方](请看日志生成结束,默认结束后会到处excel加译文写入写入ts文件)，如果有不正确的地方请手动修改第三列的翻译内容并保存excel文件，之后再译文写入。批量处理请选择对应多个ts文件所在的文件夹，建议点击“生成+翻译+导入”按钮，一键生成多国语言(批量处理的语言只有combox中写了的，其他需要单个对应百度api来处理)</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>日志打印</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_log"/>
      </item>
      <item>
       <widget class="QPushButton" name="clearLog">
        <property name="text">
         <string>清空日志</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
